docs/guide/cli.md

```diff
@@ -36,6 +36,10 @@ Useful to run with [`lint-staged`](https://github.com/okonet/lint-staged) or wit
 vitest related /src/index.ts /src/hello-world.js
 ```
 
+### `vitest clearCache`
```

Comment:
```suggestion
### `vitest purge`
```

I am not a fan of pascal case in CLI. Maybe `purge`, WDYT?

Reply:
Why `purge` tho? Don't see any connection to cache. We can do `clear-cache`, I guess.

Reply:
Just want to find a single word to represent it. As for reference, here is what package managers are doing:

```ts
npm cache clean --force
yarn cache clean
pnpm store prune
```

And for Vite: https://vitejs.dev/guide/dep-pre-bundling.html#file-system-cache

/cc @patak-dev in case we are also going to add a clear cache command for Vite, I think we better align the commands

Reply:
We can do `vitect clean`, and pass `cache` as an argument. In future we might add option to clear only some specific caches.

Reply:
I've updated to proposed solution. What do you think, @antfu, @patak-dev?

Reply:
I don't like solution with `vitest cache <action>`, because I think we shouldn't all anything done to cache except clearing.

Reply:
`clean` looks good to me

Reply:
This is something we should also clean in Vite, but whenever we discuss in a meeting, 10 options appear and we get stuck deciding 👀
For vite, we now ended up moving force to optimizeDeps.force, hinting that there won't be a global cache object but each feature may have its own? 

I think that the global cache config is good for Vitest though (and probably also we should move cacheDir to cache.dir like you have in the PR)

I thought while reading the PR that you could piggyback on vite here and use `${cacheDir}/vitest`... but probably a good idea to keep them separate as there is a log of people used to do rm -rf ${cacheDir}

About the command, vitest clean sounds good, I agree with you that we may now want to follow the yarn cache clean here
I would prefer a flag instead of a command though, as it is very useful to run something while clearing the cache (like --force):
```
vitest run --coverage --clean
```
And maybe by default if no params are passed it means to clean everything?

Reply:
> I would prefer a flag instead of a command though, as it is very useful to run something while clearing the cache (like --force):

You have a point! Maybe we can do something like

```
vitest run --no-cache
```

I don't like `--force`, because Vite already uses it and it may be confusing. And `--clean` or `--clean` is not expressive enough for me 👀

Reply:
Although the fact that `--no-cache` removes cache is also not apparent 🤔

Reply:
I guess we can just do `vitest run --clear-cache`

Reply:
I actually wanted `--no-cache` for Vite too but it isn't clear if you start partitioning the cache. I think that `vitest run --clean` is clear enough, it is something you use a lot so it is better if it is a single word

Reply:
> it is something you use a lot so it is better if it is a single word

Hardly disagree with using a lot - I don't know why you would even use it in Vitest, only to clear malformed cache maybe.

Also single words only _look_ good, but confuse a lot. What `--clean` means? What happens when you call it?

Reply:
I use `--force` a lot because ya, I'm debugging Vite cache itself 👀 
So I agree with you that normally isn't something that final users will be reaching a lot. I'm fine with `--clearCache` (maybe good to check how it will look later for `--clearCache="parts of it"`)

Reply:
> maybe good to check how it will look later

It will look something like this:
```
vitest run --clearCache results
```
We use cac's notation

Reply:
@antfu what do you think?

Reply:
I will remove command for now, since I want this feature to be released today. Command will be in another PR.